android-json-form-wizard/src/main/java/com/vijay/jsonwizard/fragments/JsonFormFragment.java

```diff
@@ -107,13 +107,33 @@ public boolean onOptionsItemSelected(MenuItem item) {
             presenter.onBackClick();
             return true;
         } else if (item.getItemId() == R.id.action_next) {
-            presenter.onNextClick(mMainView);
-            return true;
+            return next();
         } else if (item.getItemId() == R.id.action_save) {
```

Comment:
Cool. Android does not support using Android resource IDs (`R.id.action_next` for instance) in switch statements.